server/profileGet.php

```diff
@@ -0,0 +1,31 @@
+<?php
+$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->hasPermission('can_use_admin_tools')) {
+  header("Location: /unauthorized");
+  die();
+}
+require_once 'config.php';
+getProfile($USER->getUserId());
+function getProfile($data) {
```

Comment:
If this method is only taking in the userId, you should probably just call it $userId